### PR TITLE
Fixed scrolling issue in editor

### LINKED ISSUE #658: Add padding and scroll options to the editor
Right now the editor does not look so real.
By adding default padding and giving a scroll option it would feel more real.

### PR DESCRIPTION
**Pull request by Rishab-j**
Now the user would be able to scroll the output field of the editor if the input data is large.

Closes #658

### CODE REVIEW

File: src/pages/Editor/sections/OutputComponent/Output.js [hunk 1]
```diff
@@ -57,6 +57,7 @@ const OutputComponent = () => {
               color: `${editContext.bodyValues.bodyColor}`,
               width: `${editContext.bodyValues.bodyWidth}pc`,
               letterSpacing: `${editContext.bodyValues.bodyLetterSpace}px`,
+              overflowY : `scroll`,
```
Comment (iamdarshshah) [inline src/pages/Editor/sections/OutputComponent/Output.js:60]:
```suggestion
              overflowY : 'scroll',
```
Why're you making use of template literals, if you don't want to embed expression?

Comment (Rishab-j) [reply src/pages/Editor/sections/OutputComponent/Output.js]: Will keep it in my mind in the future, Thank You.

**Review by anushbhatia — CHANGES_REQUESTED** [review]
Add the same in the title too.

**Review by iamdarshshah — APPROVED** [review]

**Comment by anushbhatia** [comment]
Great Work @Rishab-j ✨✨

**Comment by Rishab-j** [comment]
Thank You @anushbhatia

**Pull request merged**